pkgs/by-name/ra/radicle-httpd/package.nix

```diff
@@ -16,9 +16,11 @@ rustPlatform.buildRustPackage rec {
   env.RADICLE_VERSION = version;
 
   # You must update the radicle-explorer source hash when changing this.
-  src = fetchgit {
-    url = "https://seed.radicle.xyz/z4V1sjrXqjvFdnCUbxPFqd5p4DtH5.git";
-    rev = "refs/namespaces/z6MkireRatUThvd3qzfKht1S44wpm4FEWSSa4PRMTSQZ3voM/refs/tags/v${version}";
+  src = fetchFromRadicle {
+    seed = "seed.radicle.xyz";
+    repo = "z4V1sjrXqjvFdnCUbxPFqd5p4DtH5";
+    node = "z6MkireRatUThvd3qzfKht1S44wpm4FEWSSa4PRMTSQZ3voM";
+    tag = "v${version}";
     hash = "sha256-9rJH4ECqOJ9wnYxCbEFHXo3PlhbPdeOnF+Pf1MzX25c=";
     sparseCheckout = [ "radicle-httpd" ];
```

Comment:
sparseCheckout changes the hash, right? Then I would recommend writing the hash last to make that obvious.